client/src/Components/Origin.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState, useEffect } from 'react';
+
+function Origin (props) {
+  const [topCountries, setCountries] = useState([]);
+  const [topCities, setCities] = useState([]);
+
+  useEffect(() => {
+    getTopCountries(props.locations);
+  }, [props.locations]);
+
+  const getTopCountries = (countries) => {
+    let filteredCountries = [], sortedCountries = [];
```

Comment:
what does filtered countries represent? Is filtered an effective name for this object.